### DISCUSSION TITLE
First Three Fourths

### DISCUSSION THREAD

**Discussion by xguiboy** [rxhanson/Rectangle#1141] [General]
Feature ideas are now kept in the [discussions tab](https://github.com/rxhanson/Rectangle/discussions). Please create an issue there, not here.

i would like to know, if instead of First Three Fourths, i could add a First Four Fifths, because it is a little bigger and i can see 2 icons rolls in the desktop, thanks

**Comment by rxhanson** [comment]
Not in Rectangle, but you can in [Rectangle Pro](https://rectangleapp.com/pro) (paid) by creating a custom shortcut.
